src/sentry/api/endpoints/organization_discover_query.py

```diff
@@ -13,14 +13,20 @@
 )
 
 from sentry.api.serializers.rest_framework import ListField
-from sentry.api.bases.organization import OrganizationDiscoverPermission
+from sentry.api.bases.organization import OrganizationPermission
 from sentry.api.bases import OrganizationEndpoint
 from sentry.models import Project, ProjectStatus, OrganizationMember, OrganizationMemberTeam
 from sentry.utils import snuba
 from sentry import roles
 from sentry import features
 
 
+class OrganizationDiscoverQueryPermission(OrganizationPermission):
+    scope_map = {
+        'POST': ['org:read', 'project:read'],
```

Comment:
Is there no GET now?

Reply:
I realised the other endpoint can use standard OrganizationPermission, this one is just specific to discover queries (post)